tests/Pyxsim/testers.py

```diff
@@ -0,0 +1,115 @@
+import Pyxsim
```

Comment:
I would like to get the Pyxsim items moved somewhere else eventually, but I think it is reasonable to move them here temporary.